Remotion/ObjectBinding/Web/UI/Controls/BocList.cs

```diff
@@ -3432,10 +3431,41 @@ public WebMenuItemCollection ListMenuItems
     [Category ("Menu")]
     [Description ("The width reserved for the menu block.")]
     [DefaultValue (typeof (Unit), "")]
+    [Obsolete ("Use " + nameof(MenuBlockMinWidth) + " and " + nameof(MenuBlockMaxWidth) + " instead.", true)]
     public Unit MenuBlockWidth
     {
-      get { return _menuBlockWidth; }
-      set { _menuBlockWidth = value; }
+      get => throw new NotSupportedException ($"Use {nameof (MenuBlockMinWidth)} and {nameof (MenuBlockMaxWidth)} instead.");
+      set => throw new NotSupportedException ($"Use {nameof (MenuBlockMinWidth)} and {nameof (MenuBlockMaxWidth)} instead.");
+    }
+
+    /// <inheritdoc />
+    [Category ("Menu")]
+    [Description ("The minimum width reserved for the menu block.")]
+    [DefaultValue (typeof (Unit), "")]
+    public Unit MenuBlockMinWidth
+    {
+      get => Unit.Parse (Style["--boclist-menublock-minimum-width"]);
+      set
+      {
+        var valueAsString = value.ToString();
+        if (!string.IsNullOrEmpty (valueAsString))
+          Style["--boclist-menublock-minimum-width"] = valueAsString;
+      }
```

Comment:
interesting solution. 

please make the getter a regular property for symmetry. 
the else-branch in the settings needs a remove-value statement.

Also, I'm not sure this doesn't conflict with the designer, i.e. this code may result in the style attribute rendering the css-variable back into the aspx markup during an update. So, yes, I do appreciate the elegance, but we shoudn't use the style property for this.